lib/Sema/CSBindings.cpp

```diff
@@ -361,17 +359,6 @@ ConstraintSystem::getPotentialBindingForRelationalConstraint(
     type = OptionalType::get(type);
   }
 
-  // If the type we'd be binding to is a dependent member, don't try to
-  // resolve this type variable yet.
-  if (type->is<DependentMemberType>()) {
-    if (!ConstraintSystem::typeVarOccursInType(typeVar, type,
```

Comment:
Just to confirm: The occurs check is being handled by the normal bindings path?

Reply:
Yes, it's verified separately from dependent member types.